intro_to_rl.ipynb

```diff
@@ -0,0 +1,3249 @@
+{
```

Comment:
<p>Should we not print the hint here, instead of showing "hint..." perhaps? Might not be clear that hint is in a hidden docstring.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
removed this section